pay-api/src/pay_api/models/invoice.py

```diff
@@ -140,6 +141,7 @@ class Invoice(Audit):  # pylint: disable=too-many-instance-attributes
     payment_account = relationship("PaymentAccount", lazy="joined")
     references = relationship("InvoiceReference", lazy="joined")
     corp_type = relationship("CorpType", foreign_keys=[corp_type_code], lazy="select", innerjoin=True)
+    partial_refunds = relationship("RefundsPartial", lazy="joined")
```

Comment:
There's probably a way to toggle this relationship on demand

Reply:
code removed